src/webhook_service/terraform/dynamodb.tf

```diff
@@ -8,6 +8,40 @@ resource "aws_dynamodb_table" "webhook" {
     type = "S"
   }
 
+  attribute {
+    name = "sequence_number"
+    type = "N"
+  }
+
+  attribute {
+    name = "webhook_type"
+    type = "S"
+  }
+
+  # The order of sequence_number is the same as the order of ascending order of created_at
+  # So sorting by sequence_number is the same as sorting by created_at, we only need to use SequenceNumberIndex to sort here.
+  global_secondary_index {
+    name               = "SequenceNumberIndex" # Updated GSI name
+    hash_key           = "webhook_type"        # Partition key
+    range_key          = "sequence_number"     # Sort key
+    projection_type    = "ALL"                 # Include all attributes in the GSI
+  }
+
```

Comment:
Please enable the PITR for prod env, and enable the deletion protection

For example:
```HCL
  deletion_protection_enabled = var.enable_deletion_protection_for_dynamodb_table

  point_in_time_recovery {
    enabled = var.enable_pitr
  }
  ```

Reply:
Delete protection and PITR have been added to both `webhook table` and `webhook_total_count` table
```
  deletion_protection_enabled = var.enable_deletion_protection_for_dynamodb_table  

  point_in_time_recovery {  
    enabled = var.enable_pitr  
  }  
 ```
Take a look at the [dynamodb.tf](https://github.com/aws-educate-tw/aws-educate-tpet-backend/blob/SCRUM-157-list-webhooks/src/webhook_service/terraform/dynamodb.tf)